### PR TITLE
Feature/time measurable

### PR DESCRIPTION
**Pull request by kazuooooo**
#5 #16 
* Add Time Measurable Function using DecoratorPattern (Also sample Octcatable tmp)
https://medium.com/@dan_abramov/mixins-are-dead-long-live-higher-order-components-94a0d2f9e750

* Add circleci.yml

hello_electron_react__ _comparing_develop___feature_time_measurable_ _kazuooooo_hack

### CODE REVIEW

File: app/components/Task.js [hunk 1]
```diff
@@ -5,12 +5,31 @@ import ContentAdd from 'material-ui/svg-icons/content/add';
...
-import { ListItem } from 'material-ui/List';
 import PropTypes from 'prop-types';
 
+import TimeMeasurable from './task_decorators/TimeMeasurable';
+import Octcatable from './task_decorators/Octcatable';
+
 import styles from './Task.css';
 
 class Task extends Component {
+  static propTypes = {
```
Comment (matart15) [inline app/components/Task.js:16]: if we are using propTypes then we might not need flow.

Comment (matart15) [reply app/components/Task.js:16]: also flow requires `// @flow` at start of every files

File: app/components/Task.js [hunk 2]
```diff
@@ -130,31 +132,13 @@ class Task extends Component {
...
-    complete: PropTypes.bool,
-    childTasks: PropTypes.array
-  })
-};
-Task.defaultProps = {
-  task: []
-};
-export default Task;
+// TODO: control decoration with setting
+export default Octcatable(TimeMeasurable(Task));
```
Comment (kazuooooo) [inline app/components/Task.js:143]: Task can be decorated like this

File: app/components/Task.js (hunk 1 above)
Comment (kazuooooo) [reply app/components/Task.js:16]: oh!

File: app/store/configureStore.dev.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ const configureStore = (initialState) => {
   // restore state
   try {
     // tmp
-    localStorage.state = JSON.stringify([]);
+    // localStorage.state = JSON.stringify([]);
     initialStateFromLocalState = JSON.parse(localStorage.state);
```
Comment (matart15) [inline app/store/configureStore.dev.js]: I think this line gives can't parse undefined error

File: package.json [hunk 4]
```diff
@@ -192,9 +191,11 @@
     "font-awesome": "^4.7.0",
     "history": "^4.6.3",
     "material-ui": "^0.19.4",
-    "react": "^15.6.1",
+    "moment": "^2.20.1",
+    "react": "^15.6.2",
```
Comment (matart15) [inline package.json:195]: not related to this pull request. But Realized we are using old version of react.

**Pull request merged**